### PR TITLE
Added some help text for meteor test-app

### PR DESCRIPTION
**Pull request by tmeasday**
First pass at this. TBH I'm not totally sure what goes in here. Am I right to call out to the guide? Should we do this in other spots of the help?

### CODE REVIEW

File: tools/cli/help.txt [hunk 1]
```diff
@@ -532,7 +532,65 @@ Options:
...
+       meteor test-app --unit [options]
+       meteor test-app --integration [options]
+       
+Runs tests against the application, either in *unit* or *integration* 
+test mode. meteor test-app will start a special app based on a test driver
+which handles the task of running tests and displaying the results in the 
+browser when you visit it.
+
+In unit test mode, no files in your application are eagerly loaded, aside
+from test files (in you tests/ directory or named *.tests.*). You can
```
Comment (avital) [inline tools/cli/help.txt]: FWIW, Right now it also finds files named `tests.*` or `test.*` or `*.test.*`. (We can change that if we want)

File: tools/cli/help.txt [hunk 2]
```diff
@@ -532,7 +532,65 @@ Options:
...
+import your app's modules from within your tests and test them in isolation.
+
+In integration test mode, your app is loaded as usual, and your tests can
```
Comment (avital) [inline tools/cli/help.txt]: Maybe "your app is loaded as usual, and then made hidden, ..."?

File: tools/cli/help.txt [hunk 3]
```diff
@@ -532,7 +532,65 @@ Options:
...
+                    in a browser or in console logs.
+
+  --ios,            Run tests in an emulator or on a mobile device. All of
+  --android,        the tests for client and server will run in addition to
+  --ios-device,     mobile-specific tests.
+  --android-device
+  --test-app-path   Set the directory in which to create a temporary app used
+                    for tests. Defaults to the system's temporary directory,
+                    usually /tmp.
+  --velocity        [Experimental] Execute tests using phantomjs and exit.
```
Comment (avital) [inline tools/cli/help.txt]: Is that really the only difference? (If so, wow.)

**Comment by avital** [comment]
This is so cool! I never knew about some of those options, I guess we just got them for free from test-packages?

**Comment by avital** [comment]
And I think calling out the guide is great! Yes, we should do it for more commands!

**Comment by avital** [comment]
The help should explain the default behavior (which is `--integration`). Or, we could require passing an argument, that may be better?

**Comment by tmeasday** [comment]
Please take another go-round

**Comment by avital** [comment]
LGTM

**Pull request merged**